pombola/south_africa/static/sass/_south-africa_overrides.scss

```diff
@@ -1737,6 +1737,21 @@ a.alphabet-links__link {
     }
 }
 
+li, div {
+    span.inline-search {
+         padding: 0.33em;
+         display: inline-block;
```

Comment:
I think these two lines should probably be indented 4 spaces rather than 5, to be consistent with the rest of the file :)